cfme/configure/configuration/server_settings.py

```diff
@@ -40,6 +42,59 @@
 }
 
 
+class ServerRolesView510(View):
+    """ Class represents Server Roles Form """
+    automate = BootstrapSwitch(name="server_roles_automate")
+    ems_metrics_coordinator = BootstrapSwitch(name="server_roles_ems_metrics_coordinator")
+    ems_metrics_collector = BootstrapSwitch(name="server_roles_ems_metrics_collector")
+    ems_metrics_processor = BootstrapSwitch(name="server_roles_ems_metrics_processor")
+    cockpit_ws = BootstrapSwitch(name="server_roles_cockpit_ws")
+    database_operations = BootstrapSwitch(name="server_roles_database_operations")
+    embedded_ansible = BootstrapSwitch(name='server_roles_embedded_ansible')
+    event = BootstrapSwitch(name="server_roles_event")
+    git_owner = BootstrapSwitch(name="server_roles_git_owner")
+    notifier = BootstrapSwitch(name="server_roles_notifier")
+    ems_inventory = BootstrapSwitch(name="server_roles_ems_inventory")
+    ems_operations = BootstrapSwitch(name="server_roles_ems_operations")
+    reporting = BootstrapSwitch(name="server_roles_reporting")
+    scheduler = BootstrapSwitch(name="server_roles_scheduler")
+    smartproxy = BootstrapSwitch(name="server_roles_smartproxy")
+    smartstate = BootstrapSwitch(name="server_roles_smartstate")
+    user_interface = BootstrapSwitch(name="server_roles_user_interface")
+    web_services = BootstrapSwitch(name="server_roles_web_services")
+    websocket = BootstrapSwitch(name="server_roles_websocket")
+
+    default_smart_proxy = Text(
+        "//label[contains(text(), 'Default Repository SmartProxy')]/following-sibling::div")
+
+
+class ServerRolesView511(View):
```

Comment:
Hey lots of repetition... lets use view inheritance like
```
class BaseServerRolesView(View):
    """ Class represents Server Roles Form """
     # all common things for both view

class ServerRolesView510(BaseServerRolesView):
    websocket = BootstrapSwitch(name="server_roles_websocket")

    
class ServerRolesView511(BaseServerRolesView):
    internet_connectivity = BootstrapSwitch(name="server_roles_internet_connectivity")
    remote_console = BootstrapSwitch(name="server_roles_remote_console")
```
I think it will reduce repetition.

Reply:
Won't that affect the ordering of widgets in the view? I thought the ordering was important for proper use of views in widgetastic.

Reply:
Yes, widgets order is so important in case of `dependent widgets`. I don't found these are dependent widgets. is it?